File: app/src/main/java/com/example/projectappdoctruyen/Activity/FeedBackActivity.java
package com.example.projectappdoctruyen.Activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.content.Intent;
import android.os.Bundle;
import android.util.Log;
import android.view.View;

import com.example.projectappdoctruyen.Model.Story;
import com.example.projectappdoctruyen.R;
import com.google.android.gms.ads.AdRequest;
import com.google.android.gms.ads.AdView;
import com.google.android.gms.ads.MobileAds;

public class FeedBackActivity extends AppCompatActivity {
    private AdView adView;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_feed_back);
        adView = findViewById(R.id.adView);




    }


}

File: app/src/main/java/com/example/projectappdoctruyen/Activity/FeedBackActivity.kt
package com.example.projectappdoctruyen.Activity

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.example.projectappdoctruyen.R
import com.google.android.gms.ads.AdView

class FeedBackActivity : AppCompatActivity() {
    private var adView: AdView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_feed_back)
        adView = findViewById(R.id.adView)
    }
}